blockchain/chain/src/store/chain_store.rs

```diff
@@ -109,7 +109,7 @@ where
         cs
     }
 
-    /// Sets heaviest tipset within ChainStore and store its tipset cids under HEAD_KEY
+    /// Sets heaviest tipset within `ChainStore` and store its tipset CIDs under `HEAD_KEY`
```

Comment:
Maybe we can re-word the docs to talk about keys rather than CIDs.
```suggestion
    /// Sets heaviest tipset within `ChainStore` and store its tipset keys under `HEAD_KEY`
```

Reply:
Good idea.